<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>智能交通咨询系统 - 增强版</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="mainLayout">
    <item>
     <widget class="QWidget" name="inputPanel">
      <layout class="QVBoxLayout" name="inputLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_shortest">
         <property name="title">
          <string>最短路径查询</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_shortest">
          <item row="0" column="0">
           <widget class="QLabel" name="label_start">
            <property name="text">
             <string>起点</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="startcity">
            <property name="placeholderText">
             <string>请输入起点城市</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_end">
            <property name="text">
             <string>终点</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="endcity">
            <property name="placeholderText">
             <string>请输入终点城市</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_pass">
            <property name="text">
             <string>绕过城市</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="passcity">
            <property name="placeholderText">
             <string>无</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_algo">
            <property name="text">
             <string>使用算法</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="comboBox">
            <item>
             <property name="text">
              <string>Dijkstras算法</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Fload算法</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_shortest">
            <property name="text">
             <string>最短距离</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="shortest"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_path">
            <property name="text">
             <string>路径</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="path"/>
          </item>
          <item row="6" column="0" colspan="2">
           <widget class="QPushButton" name="search">
            <property name="text">
             <string>查询</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_allpaths">
         <property name="title">
          <string>所有可行路径查询</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_allpaths">
          <item row="0" column="0">
           <widget class="QLabel" name="label_cityn">
            <property name="text">
             <string>请输入最多经过的城市数</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="cityn">
            <property name="placeholderText">
             <string>如10</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QPushButton" name="searchn">
            <property name="text">
             <string>查询</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_kpath">
         <property name="title">
          <string>第K短路径查询</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_kpath">
          <item row="0" column="0">
           <widget class="QLabel" name="label_kvalue">
            <property name="text">
             <string>请输入K值</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="kvalue">
            <property name="placeholderText">
             <string>如3</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_kshortest">
            <property name="text">
             <string>最短距离</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="kshortest"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_kpath">
            <property name="text">
             <string>路径</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="kpath"/>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>查询</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>600</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>这里显示所有路径、结果和提示信息</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
